content/docs/1.7.0/advanced-resources/os-distro-specific/container-optimized-os-support.md

```diff
@@ -0,0 +1,69 @@
+---
+title:  Container-Optimized OS Support
+weight: 5
+---
+
+## Requirements
+
+You must meet the following requirements before installing Longhorn on a Container-Optimized OS (COS) cluster.
+
+**Note:** Longhorn currently supports Container-Optimized OS only when used as the base image for Google Kubernetes Engine (GKE). This limitation is due to GKE comes with a pre-configured Kubernetes environment. For self-prepared Kubernetes environment or Kubernetes provisioned with other orchestrators, different strategies my be required as the provided method may be ineffective.
+
+### Dependencies Configuration and Installation
+
+The Container-Optimized OS does not include a package manager and doesn't support running non-containerized application. Additionally, its root filesystem is mounted as read-only, which poses a challenge for IO operations.
+
+In GKE, Kubernetes tackes these constraints by housing necessary dependencies in a chroot environment (/home/kubernetes/containerized_mounter/rootfs) and mounting directories within it, enablind the execution of required tasks.
+
+Longhorn provides a GKE COS node agent, leveraging GKE Kubernetes solutions to configure an run necessary dependencies. This agent is responsible for:
+
+- Mounting the Longhorn data path.
+- Loading the Kernel module.
+- Installing and running the iSCSI daemon.
```

Comment:
```suggestion
### Dependencies, Installation, and Configuration

The Container-Optimized OS does not include a package manager and does not allow non-containerized applications to run. Additionally, its root filesystem is mounted as read-only, which poses a challenge for IO operations.

In GKE, Kubernetes tackes these constraints by housing necessary dependencies in a chroot environment (`/home/kubernetes/containerized_mounter/rootfs`) and mounting directories within it, enabling the execution of required tasks.

Longhorn provides a GKE COS node agent, which leverages GKE Kubernetes solutions to configure and run necessary dependencies. This agent is responsible for the following operations:

- Mounting the Longhorn data path
- Loading the kernel module
- Installing and running the iSCSI daemon
```

Reply:
> ### Dependencies, Installation, and Configuration
I am not sure if we should separate these words. It seems to imply three distinct tasks, but actually, they are all related to dependencies (data path, module, package..). Maybe we could consider removing this title and integrating the contexts under the `Requirement` section instead. What do you think?

Reply:
Oh, you meant "Installation and Configuration of Dependencies". The new section heading works.